D-Upchurch/front-end-capstone
# Nav Bar

# Description
Nav Bar in and working, pages do not render as the components haven't been build out yet.

## Type of change


# Testing Instructions
Clone repo, fetch and check into dcu-nav-bar branch. Run 'npm start' from root project directory in terminal, in a new tab navigate to the api directory of the project and run the json server. Go to your browser, navigate to the page, register a user, navigate to http://localhost:3000/ to see nav bar
# Checklist:
